model/patch/cli_intent.go

```diff
@@ -97,6 +97,9 @@ type cliIntent struct {
 	RepeatFailed bool `bson:"repeat_failed"`
 	// RepeatPatchId uses the given patch to reuse the task/variant definitions
 	RepeatPatchId string `bson:"repeat_patch_id"`
+
+	// LocalModuleIncludes is only used to include local module changes
```

Comment:
I understand why this is true for patches, but not why its true for intents. We store intents in the DB so that if the patch intent processor has an issue, we can still process the patch. Are you sure we aren't using this functionality?

Reply:
If that's true though, can we make this private?

Reply:
that's my bad. saving in db again